packages/gestalt/src/Link/Link.js

```diff
@@ -53,8 +54,21 @@ export default class Link extends React.Component<Props, State> {
   };
 
   render() {
-    const { children, inline = false, target = null, href } = this.props;
-    const rel = target === 'blank' ? 'noopener noreferrer' : null;
+    const {
+      children,
+      inline = false,
+      nofollow = false,
+      target = null,
+      href
+    } = this.props;
+    const relAttributes = [];
+    if (target === 'blank') {
+      relAttributes.push('noopener', 'noreferrer');
+    }
+    if (nofollow) {
+      relAttributes.push('nofollow');
+    }
+    const rel = relAttributes.join(' ');
```

Comment:
@istian11 We want to make sure `rel` is `null` instead of an empty string when nothing needs to be set.